tvb_library/tvb/basic/neotraits/info.py

```diff
@@ -156,3 +177,16 @@ def trait_object_repr_html(self):
     result += ['</tbody></table>']
 
     return '\n'.join(result)
+
+
+def prepare_html(doc):
+    # type: (str) -> str
+    """
+    Create html (that can be further enhanced by MathJax) from the description received as parameter
+    """
+    try:
+        html = sphinxify.sphinxify(doc, "sphinxify_dir", "html")
```

Comment:
If this remains this short maybe put the `sphinxify` call inside `trait_object_repr_html ` function directly and avoid having a supplementary function here?
It is not used in other places, right ?